tests/foreman/cli/test_import.py

```diff
@@ -315,14 +341,56 @@ def test_import_users_default(self, data):
         })
         os.remove(new_dataset)
         ssh.command(
-            'rm -rf ${{HOME}}/.transition_data {0} {1}'
-            .format(pwdfile, tmp_dir)
+            'rm -rf ${{HOME}}/.transition_data {}'
+            .format(pwdfile)
         )
         self.assertEqual(
             ssh_import.stdout,
             [u'Summary', u'  Created 3 users.', u'']
         )
 
+    @data(*positive_import_user_data())
+    def test_reimport_users_default(self, data):
```

Comment:
Nice improvement. Can you also fix the `redefined-outer-name` warnings?

    W:169,39: Redefining name 'data' from outer scope (line 10) (redefined-outer-name)
    W:208,41: Redefining name 'data' from outer scope (line 10) (redefined-outer-name)
    W:249,41: Redefining name 'data' from outer scope (line 10) (redefined-outer-name)
    W:277,30: Redefining name 'data' from outer scope (line 10) (redefined-outer-name)
    W:322,40: Redefining name 'data' from outer scope (line 10) (redefined-outer-name)
    W:357,42: Redefining name 'data' from outer scope (line 10) (redefined-outer-name)
    W:412, 8: Redefining name 'data' from outer scope (line 10) (redefined-outer-name)

The fix is as simple as renaming `data` to something else, like `test_data`.